app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -20,20 +20,20 @@ def idv_failure(result)
           @flow.analytics.track_event(
             Analytics::THROTTLER_RATE_LIMIT_TRIGGERED,
             throttle_type: :idv_resolution,
-            step_name: self.class.class_name,
+            step_name: self.class,
```

Comment:
we can do `.class.name` (which Class inherits from Module): https://ruby-doc.org/core-2.5.3/Module.html#method-i-name

```suggestion
            step_name: self.class.name,
```